src/include/daos/drpc_modules.h

```diff
@@ -37,6 +37,7 @@ enum drpc_module {
 	DRPC_MODULE_TEST		= 0,	/* Reserved for testing */
 	DRPC_MODULE_SECURITY_AGENT	= 1,
 	DRPC_MODULE_MGMT_SERVER		= 2,
+	DRPC_MODULE_SRV			= 3,	/* daos_server */
```

Comment:
why do we have this in addition to #2 surely they are the same thing

Reply:
I'm not sure... It seems DRPC_MODULE_MGMT_SERVER is for Go control/server -> C mgmt. DRPC_MODULE_SRV is for C iosrv -> Go control/server. Let me know what the rule is in this case.

Reply:
I think it should be a separate module personally. It seems like this module is for infrastructure communications. Maybe it needs a different name? DRPC_MODULE_COMM?

Reply:
I thought we could probably merge mgmt -> iosrv at some point given a lot of the code in src/mgmt will go away when we move dmg tool into Go. then just have a single module to handle duplex communication between iosrv/Go control/server. I am probably mistaken though.

Reply:
> It seems like this module is for infrastructure communications.

Same feeling here.

> I thought we could probably merge mgmt -> iosrv at some point given a lot of the code in src/mgmt will go away when we move dmg tool into Go. then just have a single module to handle duplex communication between iosrv/Go control/server. I am probably mistaken though.

The boundary between mgmt and iosrv seems blur to me from time to time. However, I'd say if we could refrain the urge to restructure until the new dmg, the new bootstrap, the new format process, etc., are mostly in place; personally, the restructure frequency in the control plane code feels a bit too high recently.